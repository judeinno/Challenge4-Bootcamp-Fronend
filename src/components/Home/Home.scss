@import url('https://fonts.googleapis.com/css?family=Dosis:400,500,600');

.home {
    margin-top: 5%;
    font-family: 'Dosis', sans-serif;
}
