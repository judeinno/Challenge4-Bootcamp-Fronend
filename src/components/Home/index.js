import React from 'react';
import { BrowserRouter, Route } from 'react-router-dom';
import Questions from '../../containers/AllQuestionsPage';
import './Home.scss';

const Home = () => (
  <BrowserRouter>
    <div className="container home">
      <h3>Questions</h3>
      <Route path="/" component={Questions} />
    </div>
  </BrowserRouter>
);

export default Home;
