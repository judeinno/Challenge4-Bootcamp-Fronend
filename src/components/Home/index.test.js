import React from 'react';
import { shallow } from 'enzyme';
import Home from './index';

it('testing Home', () => {
  const HomeComponent = shallow(
    <Home />,
  );
  expect(HomeComponent).toMatchSnapshot();
});
