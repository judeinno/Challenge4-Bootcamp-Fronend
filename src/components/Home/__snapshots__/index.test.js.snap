// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Home 1`] = `
<BrowserRouter>
  <div
    className="container home"
  >
    <h3>
      Questions
    </h3>
    <Route
      component={[Function]}
      path="/"
    />
  </div>
</BrowserRouter>
`;
